test/e2e/network/netpol/kubemanager.go

```diff
@@ -181,12 +185,22 @@ func (k *kubeManager) probeConnectivity(args *probeConnectivityArgs) (bool, stri
 	}
 
 	commandDebugString := fmt.Sprintf("kubectl exec %s -c %s -n %s -- %s", args.podFrom, args.containerFrom, args.nsFrom, strings.Join(cmd, " "))
-	stdout, stderr, err := k.executeRemoteCommand(args.nsFrom, args.podFrom, args.containerFrom, cmd)
-	if err != nil {
-		framework.Logf("%s/%s -> %s: error when running command: err - %v /// stdout - %s /// stderr - %s", args.nsFrom, args.podFrom, args.addrTo, err, stdout, stderr)
-		return false, commandDebugString, nil
+	probeRetryDelay := args.retryDelay
+	for i := 0; i < args.maxAttempts; i++ {
+		stdout, stderr, err := k.executeRemoteCommand(args.nsFrom, args.podFrom, args.containerFrom, cmd)
+		if err != nil {
+			framework.Logf("attempt %d: %s/%s -> %s: error when running command: err - %v /// stdout - %s /// stderr - %s", i, args.nsFrom, args.podFrom, args.addrTo, err, stdout, stderr)
```

Comment:
```suggestion
			framework.Logf("probe retry %d/%d: %s/%s -> %s: error when running command: err - %v /// stdout - %s /// stderr - %s.", i, maxAttempts, args.nsFrom, args.podFrom, args.addrTo, err, stdout, stderr)
```
minor suggestion to add the "maxRetries", so we can debug at fine granularity and detect patterns in logs easily, even w/o understanding the code